From: Alexis Sukrieh <sukria@sukria.net>
Date: Thu, 26 Nov 2015 14:32:16 -0800
Subject: debian-remove-shebang

The shebang has been removed from lib/sanitize.sh. This file should not be
executed directly and is not executable. Lintian rightfully complains about the
shebang.
---
 lib/sanitize.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/sanitize.sh b/lib/sanitize.sh
index 329f4e1..2e42198 100644
--- a/lib/sanitize.sh
+++ b/lib/sanitize.sh
@@ -1,4 +1,3 @@
-#! /usr/bin/env bash
 # Copyright © 2005-2010 Alexis Sukrieh
 #
 # See the AUTHORS file for details.
